lms/envs/common.py

```diff
@@ -2139,3 +2144,5 @@
 # platform unaware of current image URLs, resulting in reverting all
 # users' profile images to the default placeholder image.
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
+PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
+PROFILE_IMAGE_MIN_BYTES = 100
```

Comment:
this is to catch empty files or files too small to reasonably contain a usable image.